packages/packages/help/package.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Do we need separated package for `help`? not sure

Reply:
NO